### PR TITLE
Version 1

### PR DESCRIPTION
**Pull request by mschottdorf**
PR to transfer all Solidworks assembly and part files to this repository. Also added:

1. Big picture hardware design history in a README file.
2. Assembly Instructions with pictures in a pdf.
3. Minor changes in the documentation to point to local files

### CODE REVIEW

**Comment by sneakers-the-rat** [comment]
the nondeterministic test behavior looks like it's coming from here, just not throwing a connectionrefused error. since it's just a pass (i think that's fine? since it's in the close method?) would it be ok to exclude from coverage? or else is it important that we figure out what would cause that exception?

**Comment by mschottdorf** [comment]
Fair enough. But it is odd, that pytest has issues with `connectionrefused`. Maybe it is a timeout issue in the VR running the tests itself.

**Review by sneakers-the-rat — COMMENTED** [review]
The changes in the PR are fine, but since our docs aren't building automatically I don't think you would have been able to check them out yet. I have just been manually building them ( https://www.peoplesvent.org/en/tmp_hardware/index.html ), but there are a number of problems with broken links and other formatting problems. I think you should probably go through the pages on the docs and make sure they look how you want to look/all the links that should be there are there. 

I  also needed to fix some problems in the required versions, should just affect the docs, but also there were some bugs in the controller docstring formatting that were throwing some worrying looking errors in the sphinx build that i tried to smooth out. Didn't change any code there.

File: _docs/hardware/assembly.md [hunk 1]
```diff
@@ -10,10 +10,9 @@ in the **Bill of Materials**:
 We'll first show you how to assemble the hardware, then the electronics,
 and finally put the two together.
 
-The entire Solidworks Assembly, with associated part files, can be found in this Google Drive: 
-https://drive.google.com/drive/folders/1YrJEOmOMZtXcHABYO0hEu0etJ7LAC9Lf?usp=sharing
+The entire Solidworks Assembly, with associated part files, can be found [here](/assets/solidworks/PVP1_Mk3.SLDASM), together with two earlier versions of the device, [Mk1](/assets/solidworks/PVP1_Mk1.SLDASM) and [Mk2](/assets/solidworks/PVP1_Mk2.SLDASM). All associated parts are collected [here](/assets/solidworks/Parts).
```
[the remark this answers is not in the document]
Comment (sneakers-the-rat) [reply _docs/hardware/assembly.md]: Made a list version with :download: directives, testing now

File: _docs/hardware/bom.rst [hunk 2]
```diff
@@ -1,8 +1,11 @@
...
 ==================
 
+The complete bill of materials is available here:
 * :download:`Bill of materials - [.csv] </assets/csv/10_27_2020_bom.csv>`
 
+The following table is a summarized version of the CSV file above, and illustrates the approximate unit cost (state 10/2020) of a single device, together with links to manufactures of the pars:
+
 .. csv-table:: PVP bill of materials
-   :file: /assets/csv/10_27_2020_bom.csv
+   :file: /assets/csv/10_27_2020_bom_short.csv
```
[the remark this answers is not in the document]
Comment (sneakers-the-rat) [reply _docs/hardware/bom.rst]: Split this up into separate tables so it's more browsable/can navigate with the TOC. May want to double check my edits?

File: _docs/assets/csv/10_27_2020_bom_short.csv [hunk 3]
```diff
@@ -0,0 +1,110 @@
+Component,Subtotal (Bulk Parts Cost) ,Example Sourcing Link,
```
[the remark this answers is not in the document]
Comment (mschottdorf) [reply _docs/assets/csv/10_27_2020_bom_short.csv]: yeah, that is the list from @juliennelachance. No changes, just reformatting

File: _docs/hardware/assembly.md [hunk 4]
```diff
@@ -10,10 +10,9 @@ in the **Bill of Materials**:
...
 
-The current assembly is named `TLA_VENTILATOR_ASSY_V2.SLDASM`.
+The current assembly is named `PVP1_Mk3.SLDASM`. We also provide more detailed build instructions in this [pdf file](/assets/PVP1 Assembly Instructions.pdf).
```
[the remark this answers is not in the document]
Comment (mschottdorf) [reply _docs/hardware/assembly.md]: yes, it is exactly the same, just in one document. Phrasing this as "PDF version of the instructions" sounds good to me.

File: _docs/hardware/bom.rst (hunk 2 above)
[the remark this answers is not in the document]
Comment (mschottdorf) [reply _docs/hardware/bom.rst]: The separate tables look alright to me. If anyone is interested in the details, they'll have to download the full CSVs anyway 👍

**Comment by mschottdorf** [comment]
Thanks for going through this all @sneakers-the-rat! Again, BOM looks alright to me. Definitely easier to read. Links to solidwork drawings work, too, and I like the "prior version" list on the assembly page. 

I'll send a note to Daniel Suo to ask if they want to add their designs in a new PR

**Comment by mschottdorf** [comment]
The PR includes:
- All solid work parts, and multiple earlier versions of PVP1.
- More documentation, such as split BOMs, and assembly instructions as pdf.
- Adding the `okh-manifest.yml` file.
- Minor changes to travis and removing redundant checks.

**Pull request merged**